src/main/java/net/minecraftforge/client/MinecraftForgeClient.java

```diff
@@ -122,10 +123,10 @@ public static void onRebuildChunk(World world, BlockPos position, ChunkRenderCac
         if (cache == null)
             regionCache.invalidate(Pair.of(world, position));
         else
-            regionCache.put(Pair.of(world, position), cache);
+            regionCache.put(Pair.of(world, position), Optional.ofNullable(cache));
     }
 
-    public static ChunkRenderCache getRegionRenderCache(World world, BlockPos pos)
+    public static Optional<ChunkRenderCache> getRegionRenderCache(World world, BlockPos pos)
```

Comment:
This is a binary breaking change. Is there a reason you are not just adding a null check on the call-site?

Reply:
Guave Cache does not allow null values as mentioned in linked issue (#7197)

Reply:
That doesn't mean this method needs to return an Optional.

Reply:
Surely I can rename this to getRegionRenderCacheOptional and make the old method link it

Reply:
Or just not change the method at all.

Reply:
Done